PigLatin/Program.cs

```diff
@@ -7,15 +8,42 @@ class Program
         public static void Main()
         {
             // your code goes here
-
+            Console.WriteLine("Welcome to the Pig Latin Translator");
+            Console.WriteLine("Please enter a word or sentence.");
+            string word = Console.ReadLine();
+            Console.WriteLine(TranslateWord(word));
             // leave this command at the end so your program does not close automatically
+            
             Console.ReadLine();
         }
         
         public static string TranslateWord(string word)
         {
             // your code goes here
+            int vowelIndex = word.IndexOfAny(new char[] {'a', 'e', 'i', 'o', 'u'});
+
+            string firstPart = word.Substring(0,vowelIndex);
+            string secondPart = word.Substring(vowelIndex);
+            word = word.ToLower();
+
+            if (vowelIndex == 0)
```

Comment:
You are `return`ing in both cases here, which means the code below will never run. Try breaking your sentence logic out into another method and calling `TranslateWord` on each word of the sentence